packages/now-routing-utils/src/index.ts

```diff
@@ -230,6 +230,7 @@ function createError(
   const errors = Array.isArray(allErrors) ? allErrors : [allErrors];
   const message = errors[0];
   const error: RouteApiError = {
+    name: 'RouteApiError',
```

Comment:
What's the purpose of `name` here?

Reply:
It's a required property on `Error`, so needed to add it in here to be able to use it with `prettyError()`. Another option would be to use `Partial<Error>` in the prettyError() definition…